### ISSUE TITLE
Can't set D1 brightness while light is off

### ISSUE THREAD

**Issue by theonlysinjin** [AlexxIT/SonoffLAN#664]
It's been about 3 months now... But I used to be able to dim my D1 while setting the switch state to off.
This was very helpful to "reset" the brightness to a less-blinding brightness, especially at night.
The flaw with the D1 is if it's on 100% brightness, and you turn it off and back on, it comes on at 100%. I've been blinded at night a few times.

So using this integration I was actually able to set the brightness on the light off event. Saving my eyes.
This seems to have broken somewhere along the line.

My automation, which used to work
```
- id: dim_lights_when_switched_off
  alias: "Dim the lights when they are switched off"

  trigger:
    - platform: state
      entity_id:
        - light.sonoff_1234567890
      to: "off"

  action:
    - service: sonoff.send_command
      data:
        device: 1234567890
        cmd: dimmable
        switch: 'off'
        brightness: 5
```

I can switch the light on like so,        
```
service: sonoff.send_command
data:
  device: 1234567890
  cmd: dimmable
  switch: 'on'
  brightness: 50
```
and I get this in the log
`{"log":"\u001b[36m2022-01-01 07:55:39 DEBUG (Sonoff_LAN) [custom_components.sonoff.sonoff_local] 1234567890 \u003c= Local3 | {'switch': 'on', 'brightness': 50, 'brightMin': 0, 'brightMax': 255, 'startup': 'off', 'mode': 0, 'rssi': -74} | 176\u001b[0m\n","stream":"stderr","time":"2022-01-01T07:55:39.063787393Z"}`

But if I run the same command with the switch as off, I get the same result... It's like it just echoes it back.
```
service: sonoff.send_command
data:
  device: 1234567890
  cmd: dimmable
  switch: 'off'
  brightness: 10
```
`{"log":"\u001b[36m2022-01-01 08:00:58 DEBUG (Sonoff_LAN) [custom_components.sonoff.sonoff_local] 1234567890 \u003c= Local3 | {'switch': 'on', 'brightness': 50, 'brightMin': 0, 'brightMax': 255, 'startup': 'off', 'mode': 0, 'rssi': -74} | 209\u001b[0m\n","stream":"stderr","time":"2022-01-01T08:00:58.486132287Z"}`

**Comment by AlexxIT** [comment]
You don't need to use `sonoff.send_command`. You need to work with light entity

**Comment by theonlysinjin** [comment]
Hey @AlexxIT , I'm not looking to dim the light while it's on, I want to dim the light while it's off.
This used to be possible shortly before I posted this issue. It used to be possible by sending it the above command.

While the light is _off_, I used to be able to adjust the brightness to a move acceptable level. Then when you turn the light on it comes on at the adjusted brightness. Almost like setting a "Default State".
